### PR TITLE
Fixes for Python 3 and documentation for Python 3 on RHEL 7

### LINKED ISSUE #252: Install with Python3
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 5.5
- Ruby: 2.0.0
- Distribution: CentOS 7.7
- Module version: master/8b828b695ce1322ee9857ff3f1dcc2eaabb4afe2 with `puppet-python 3.0.1`

## How to reproduce (e.g Puppet code you use)
How do I make the module use the python3/python36 on CentOS 7? With `manage_virtualenv`, the module uses the `'system'` version of Python - 2.7.

Thanks!

### PR DESCRIPTION
**Pull request by nmaludy**
#### Pull Request (PR) description
The current version of this module doesn't work with Python 3. 
I was able to get it working with a small change to `wsgi.py` and some external packages / configurations which i've added to the README.

#### This Pull Request (PR) fixes the following issues
Fixes #252

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -264,6 +264,74 @@ class { 'puppetboard::apache::vhost':
...
+
+``` puppet
+$mod_wsgi_package_name = 'rh-python36-mod_wsgi'
+
+package { ['python3', 'python3-pip', 'python3-devel', 'python36-virtualenv']:
+  ensure => present,
+}
+
+# configure apache
+class { 'apache':
```
Comment (bastelfreak) [inline README.md]: it would be awesome to have this in an acceptance test <3

**Comment by nmaludy** [comment]
@bastelfreak it looks like the acceptance tests have been failing for a while. I'll see if I can make another PR for those next week.

**Comment by regner** [comment]
`virtualenv_python` doesn't appear to be valid.

**Comment by regner** [comment]
Looks like maybe it should be `virtualenv_version`: https://github.com/voxpupuli/puppet-puppetboard/blob/master/manifests/params.pp#L53

**Pull request merged**